public/src/client/topic/postTools.js

```diff
@@ -125,6 +125,30 @@ define('forum/topic/postTools', [
 			});
 		});
 
+		postContainer.on('click', '.composer-submit-anonymous', function (e) {
```

Comment:
lint errors + are these from this commit?

Reply:
yep they're not from my commit.